CONTRIBUTING.md

```diff
@@ -17,6 +17,28 @@ Please don't introduce new spotbugs output.
 * `mvn spotbugs:check` to analyze project using [Spotbugs](https://spotbugs.github.io)
 * `mvn spotbugs:gui` to review report using GUI
 
+## Fetching code coverage
+
+Once you get the report code coverage, there are a couple ways you can fetch the classes that are covered in tests.
+
+This depends a lot on the operating system and environment. The following commands will open the `index.html` file in the browser.
+
+* Windows - `target/site/jacoco/index.html`
+* Linux - `xdg-open target/site/jacoco/index.html`
+* Gitpod - `cd target/site/jacoco && python -m http.server 8000`
+
+The file will have a list of package names. You can click on them to find a list of class names.
+
+The lines of the code will be covered in three different colors. Red, green, and orange.
```

Comment:
Here I do not know what the orange color means. That is why I have not added anything about it.

I was thinking of adding screenshots. But then thought against it. The instructions will help someone understand what is happening unless any of the steps change in the future.

Reply:
The orange color means that one or more of the paths through that statement have not been executed.

Consider the following statement:

```java
 String name = importantArgument != null ? importantArgument.getName() : "unknown name";
```

There are two paths in that statement, one when importantArgument is null and one when it is not null.  If the statement is never called with importantArgument as null, then the line will be colored orange to indicate that at least one path is not covered.

If you prefer a more automatic way of viewing the results, you could configure a Jenkins job that uses the JaCoCo plugin to display the results within a Jenkins job.  You can see that in a [job definition](https://github.com/MarkEWaite/docker-lfs/blob/lts-with-plugins/ref/jobs/Jenkins-Plugins-Folder/jobs/Git-Folder/jobs/git-plugin/config.xml) that I maintain.